apps/dsb-message-broker/src/auth/login-return-data.dto.ts

```diff
@@ -1,3 +1,6 @@
 export class LoginReturnDataDTO {
     token: string;
```

Comment:
Please also add Swagger annotations to the LoginReturnDataDTO object, check for examples here https://github.com/energywebfoundation/dsb/blob/master/apps/dsb-message-broker/src/channel/dto/create-channel.dto.ts

Reply:
annotations is added